homeassistant/components/homewizard/entity.py

```diff
@@ -22,8 +22,8 @@ def __init__(self, coordinator: HWEnergyDeviceUpdateCoordinator) -> None:
             manufacturer="HomeWizard",
             sw_version=coordinator.data.device.firmware_version,
             model_id=coordinator.data.device.product_type,
-            model=coordinator.data.device.product.name
-            if coordinator.data.device.product
+            model=coordinator.data.device.model_name
+            if coordinator.data.device.model_name
             else None,
```

Comment:
Can model_name be `""`? Otherwise this check does nothing?

Anyway, please wrap in parenthesis

Reply:
Good catch, is None or an actual name. Removed the if-else.